index.html

```diff
@@ -1,144 +1,129 @@
+            companyURL: 'https://google.com/',
+            w3cid: '56102'
+          },
+          {
+            name: 'Jatinder Mann',
+            company: 'Microsoft Corp.',
+            note: 'Until February 2014',
+            email: 'jmann@microsoft.com',
+            w3cid: '44357'
+          },
+          {
+            name: 'Zhiheng Wang',
+            company: 'Google Inc.',
+            note: 'Until July 2013',
+            w3cid: '43685'
+          },
+          {
+            name: 'Anderson Quach',
+            company: 'Microsoft Corp.',
+            note: 'until March 2011',
+            w3cid: '45288'
+          }
+        ],
 
-          wg: "Web Performance Working Group",
-          wgURI: "https://www.w3.org/webperf/",
-          license: 'w3c-software-doc',
-          wgPublicList: "public-web-perf",
-          subjectPrefix: "[UserTiming]",
+        wg: 'Web Performance Working Group',
+        wgURI: 'https://www.w3.org/webperf/',
+        license: 'w3c-software-doc',
+        wgPublicList: 'public-web-perf',
+        subjectPrefix: '[UserTiming]',
 
-          // This is important for Rec-track documents, do not copy a patent URI
-          // from a random document unless you know what you're doing. If in
-          // doubt ask your friendly neighbourhood Team Contact.
-          wgPatentURI:  "https://www.w3.org/2004/01/pp-impl/45211/status",
+        // This is important for Rec-track documents, do not copy a patent URI
+        // from a random document unless you know what you're doing. If in
+        // doubt ask your friendly neighbourhood Team Contact.
+        wgPatentURI:  'https://www.w3.org/2004/01/pp-impl/45211/status',
 
-          noLegacyStyle: true,
+        noLegacyStyle: true,
 
-          otherLinks: [{
-             key: 'Repository',
-             data: [{
-               value: 'We are on GitHub',
-               href: 'https://github.com/w3c/user-timing/'
-             }, {
-               value: 'Commit history',
-               href: 'https://github.com/w3c/user-timing/commits/gh-pages/index.html'
-             }, {
-               value: 'File a bug',
-               href: 'https://github.com/w3c/user-timing/issues?q=milestone:%22Level%202%22'
-             }]
-           },{
-             key: 'Mailing list',
-             data: [{
-               value: 'public-web-perf@w3.org',
-               href: 'https://lists.w3.org/Archives/Public/public-web-perf/'
-             }]
-           },{
-             key: 'Implementation',
-             data: [{
-               value: 'Can I use User Timing?',
-               href: 'http://caniuse.com/#feat=user-timing'
-             }, {
-               value: 'Test Suite',
-               href: 'http://w3c-test.org/user-timing/'
-             }, {
-               value: 'Test Suite repository',
-               href: 'https://github.com/w3c/web-platform-tests/tree/master/user-timing'
-             }]
+        otherLinks: [{

```

Comment:
Replace github related things in`otherLinks` this with: 

```
   github: "https://github.com/w3c/user-timing/"
```

Once you do that, you can also drop `edDraftURI`, as it should be automatically generated for you.

Reply:
Updated. @marcoscaceres can you confirm I did it right?